docs/admin/dbedit.md

```diff
@@ -0,0 +1,287 @@
+# db edit commands
+
+All db editing commands are provided in heketi binary. This is to highlight the restriction that db editing must be performed only when heketi server isn't running. If the commands are performed when the server is using the database, "Unable to open database: timeout" error is shown.
```

Comment:
* in heketi binary -> in the heketi binary
* when heketi server -> when the heketi server
* "Unable to open database: timeout" error is shown -> the error "Unable to open database: timeout" is shown